source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRViewController.mm

```diff
@@ -0,0 +1,127 @@
+//
+//  ACRViewController.m
+//  ACRViewController
+//
+//  Created by jwoo on 5/26/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import "ACRViewController.h"
+#import "ACRHostConfig.h"
+#import "AdaptiveCard.h"
+#import "TextBlock.h"
+#import "Image.h"
+#import <AVFoundation/AVFoundation.h>
+
+using namespace AdaptiveCards;
+
+@implementation ACRViewController
+std::shared_ptr<AdaptiveCard> adaptiveCards;
+ACRHostConfig* m_hostConfig;
```

Comment:
These are global variables, not member variables. If you want them to be member variables, you have to surround the declarations with curly braces.